### DISCUSSION TITLE
Uninstall via brew

### DISCUSSION THREAD

**Discussion by JakobSpahn** [Whisky-App/Whisky#197] [General]
When uninstalling via brew not all files get removed. For example, when reinstalling with brew all the bottles and wine is still installed. Where is the path of these binaries so I can delete myself?

Edit: For anyone with a similar issue, the path is set here
https://github.com/IsaacMarovitz/Whisky/blob/34dec070e6b196209e0dd792144767612b794312/Whisky/Utils/WineInstaller.swift#L14

And applicationSupportDirectory on my machine was in Library/Application\ Support.

**Comment by IsaacMarovitz** [comment]
The `.userDomainMask` specifies using the Library folder for your specific user, aka `~/Library`. I haven't seen anyone else run into this issue, and I'm not sure how this happened in your case.

**Comment by OscarPereiroPaymentology** [comment]
For anyone whose Whisky does not start and they think it can be related to a bad bottle, you can remove the bottles from this directory:

/Users/yourUserName/Library/Containers/Whisky

This solved my problem, a bad bottle was automatically being loaded and I needed to remove them so that I can start Whisky after a new install
